setup.sql

```diff
@@ -7,3 +7,7 @@ CREATE TABLE userdata (
 	username VARCHAR(50) PRIMARY KEY,
 	password VARCHAR
 );
+
+ALTER TABLE userdata;
+ADD COLUMN wins INT DEFAULT 0;
+ADD COLUMN losses INT DEFAULT 0;
```

Comment:
Whey are these lines down here instead of in the `CREATE TABLE` part above? Is it because we need to set their default values?

Reply:
We can move these columns into the CREATE TABLE call. No reason to modify the table after creation.
Suggestion:
```
CREATE TABLE userdata (
	username VARCHAR(50) PRIMARY KEY,
	password VARCHAR,
        wins INT DEFAULT 0,
        losses INT DEFAULT 0
);